ddtrace/contrib/celery/signals.py

```diff
@@ -33,7 +33,7 @@ def trace_prerun(*args, **kwargs):
 
     # propagate the `Span` in the current task Context
     service = config.celery['worker_service_name']
-    span = pin.tracer.trace(c.WORKER_ROOT_SPAN, service=service, resource=task.name)
+    span = pin.tracer.trace(c.WORKER_ROOT_SPAN, service=service, resource=task.name, span_type=SPAN_TYPE)
```

Comment:
This might not actually be the "worker".

With `celery` there are two ways to execute a task, either sync or async.

`task.delay()`/`task.apply_async()` (schedule to run later via worker) vs `task.apply()` (run immediately on the "producer")

We also have just `task()` which seems to side step these signals all together so they might not be getting tracked.

Reply:
@brettlangdon What do you suggest as span type ?

Reply:
What is the issue we are trying to resolve with this change?

Reply:
Also, `worker` might still be the best type to be setting here, but I just wanted to point out that `celery` is weird and has a few ways to execute these tasks. Not entirely sure when/where we want to make a distinction between `worker` and `producer`, etc

Reply:
We're trying to make sure that span_type is always set to a 'proper' value

Reply:
`worker` is likely fine here.

We can address in the future if need be.